aptos-move/e2e-tests/src/executor.rs

```diff
@@ -144,6 +144,13 @@ pub enum GasMeterType {
     UnmeteredGasMeter,
 }
 
+#[derive(Clone)]
+pub struct TimeAndGas {
```

Comment:
nit: calling it `Measurement` can be more consitent with what you call it in `e2e-benchmarks`